content/en/docs/setup/getting-started/index.md

```diff
@@ -359,6 +359,18 @@ Use the following instructions to deploy the [Kiali](/docs/ops/integrations/kial
     {{< tip >}}
     If there are errors trying to install the addons, try running the command again. There may
     be some timing issues which will be resolved when the command is run again.
+    
+    If you are using remote yaml
+    {{< text bash >}}
+    $ kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.7/samples/addons/extras/prometheus-operator.yaml 
+    $ kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.7/samples/addons/extras/zipkin.yaml
+    $ kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.7/samples/addons/grafana.yaml
+    $ kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.7/samples/addons/jaeger.yaml
+    $ kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.7/samples/addons/kiali.yaml
+    $ kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.7/samples/addons/prometheus.yaml
+    {{< /text >}}
+
+    check https://github.com/istio/istio/tree/master/samples/addons for more detail
```

Comment:
This not appropriate in this document because the earlier instructions tell the user to download the release, so nobody should be "using remote yaml".

If you want to document something like this, it should go in the samples/addons README, not here.

```suggestion
```

Reply:
I understand you concern, but the link is broken if u clicked it.
See the detail for the screenshot.

![](https://i.imgur.com/87vQ1WB.png)

Reply:
Yes, thanks, that's a separate problem that is fixed already. https://preliminary.istio.io/latest/docs/setup/getting-started/#dashboard

Reply:
One more question. 
It is okay for me to add a line to hints the user able to check `https://github.com/istio/istio/tree/master/samples/addons` for more detail.

Reply:
Sure, but instead of adding a line I think it would be cleaner to change line 352 to the following:
```
1.  Install [Kiali and the other addons]({{< github_tree >}}/samples/addons) and wait for them to be deployed.
```

Reply:
updated